dist/p5-accessibility.js

```diff
@@ -721,9 +721,7 @@ var color_lookup =
     "b" : 1,
     "name" : "magenta"
   },
-];
-
-function getElementById(id) {
+];;function getElementById(id) {
```

Comment:
where is this double ';;' coming from?

Reply:
oh! Just looked at it, the gruntfile adds a ; between 2 files as a separator. Also, the _data.js file has a ; ending it since it's basically a giant object. Thus the 2 semicolons.
Maybe we should keep the gruntfile separator and remove the one in _data.js?

Reply:
yes! sounds good